dbus-cxx-uv/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 #   You should have received a copy of the GNU General Public License
 #   along with this software. If not see <http://www.gnu.org/licenses/>.
 
-pkg_search_module( LIBUV REQUIRED libuv )
```

Comment:
libuv should be dynamically linked by default; an option to statically link would be fine.

Reply:
Ah yeah. Working on it.